### PR TITLE
ISPN-13507 Lazily load user/groups properties

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-13507

### CODE REVIEW

**Comment by tristantarrant** [comment]
Too lazy it seems :)

**Comment by tristantarrant** [comment]
Should be fixed

File: server/runtime/src/main/java/org/infinispan/server/security/realm/EncryptedPropertiesSecurityRealm.java [hunk 1]
```diff
@@ -347,30 +349,6 @@ public void registerIdentityChangeListener(Consumer<Principal> listener) {
...
       }
 
-      /**
-       * Set the {@link InputStream} to use to load the users.
-       *
-       * @param usersStream the {@link InputStream} to use to load the users.
-       * @return this {@link Builder}
-       */
-      public Builder setUsersStream(InputStream usersStream) {
-         this.usersStream = usersStream;
```
[the remark this answers is not in the document]
Comment (tristantarrant) [reply server/runtime/src/main/java/org/infinispan/server/security/realm/EncryptedPropertiesSecurityRealm.java:357]: Fixed

**Comment by ryanemerson** [comment]
`Test Result (105 failures / +102)` :scream:

**Comment by tristantarrant** [comment]
I removed one line too many :)

**Comment by tristantarrant** [comment]
@ryanemerson fixed

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

**Pull request merged**